db-console: update notification icon in alert box to use new cockroach logo

angles-n-daemons opened the pull request:
The current icon for the notification alert is the outdated cockroach db logo. To make it consistent with the rest of the system, it needed to be updated. This change both removes the old logo from the codebase as well as uses the new one from our shared icon library.

Epic: none
Fixes: #127829

Release note (ui change): changes notification alert icon

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/130333)
<!-- Reviewable:end -->

angles-n-daemons commented on pkg/ui/workspaces/db-console/src/views/shared/components/alertBox/index.tsx:
```diff
@@ -12,23 +12,27 @@ import classNames from "classnames";
+
 import "./alertbox.styl";
 
+export function alertIconFromSVG(svg: string): JSX.IntrinsicElements["img"] {
```
that's not a bad idea, I'm going to see if I can inline these all as react components instead

angles-n-daemons commented on pkg/ui/workspaces/db-console/src/views/shared/components/icons/index.tsx:
```diff
@@ -58,63 +39,102 @@ export const rightArrow = `<svg width="7px" height="10px" viewBox="0 0 7 10" ver
 
-export const criticalIcon = `<svg width="20px" height="20px" viewBox="0 0 20 20" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
-    <!-- Generator: Sketch 42 (36781) - http://www.bohemiancoding.com/sketch -->
+export const criticalIcon = (
```
These icons change the string literal svgs to inline jsx elements

angles-n-daemons commented:
bors r+

angles-n-daemons committed:
db-console: update notification icon in alert box to use new cockroach logo

The current icon for the notification alert is the outdated
cockroach db logo. To make it consistent with the rest of the
system, it needed to be updated. This change both removes the old
logo from the codebase as well as uses the new one from our
shared icon library.

Epic: none Fixes: #127829

Release note (ui change): changes notification alert icon

angles-n-daemons commented:
bors r+
